recipe/isl_declaration_macros_expanded.h

```diff
@@ -0,0 +1,2484 @@
+# 1 "isl_declaration_macros.h"
```

Comment:
How is this file referenced/used?

Reply:
See https://github.com/conda-forge/islpy-feedstock/pull/4/files#r148996684

Reply:
Thx!